learn/platform/networks/overview.md

```diff
@@ -11,7 +11,7 @@ description: 基于Moonbeam的所有MainNet和TestNet部署概述；Moonbeam是
 
  - Moonbeam: 部署于Polkadot上（2021 十二月）
  - Moonriver: 部署于Kusama上（2021 六月）
- - Moonbase Alpha: PureStake主导平行链测试网(2020 九月)
+ - Moonbase Alpha: Moonbeam 和 Moonriver 的平行链测试网(2020 九月)
```

Comment:
Same here

Reply:
It was outdated. You can reference the Overview page on the English site